sdk/keyvault/keyvault-keys/tests/list.test.ts

```diff
@@ -114,25 +114,24 @@ describe("Keys client - list keys in various ways", () => {
 
   it("list 0 versions of a non-existing key", async () => {
     let totalVersions = 0;
-    for await (let version of client.listKeyVersions(keyName)) {
+    for await (const version of client.listKeyVersions(keyName)) {
+      if (!version) break;
       totalVersions += 1;
     }
     assert.equal(totalVersions, 0, `Unexpected total versions for key ${keyName}`);
   });
 
   it("list 0 versions of a non-existing key (paged)", async () => {
     let totalVersions = 0;
-    for await (let page of client.listKeyVersions(keyName).byPage()) {
-      for (let version of page) {
-        totalVersions += 1;
-      }
+    for await (const page of client.listKeyVersions(keyName).byPage()) {
+      totalVersions += page.length;
```

Comment:
This change doesn't look correct. We're trying to test iterating through each page, and this stops testing that.

Reply:
the issue is again the unused variable (the version in this case). Any suggestion? :sun_with_face: